ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-v0.yaml

```diff
@@ -0,0 +1,153 @@
+base_images:
+  assisted-test-infra:
+    name: ocm-2.10
+    namespace: edge-infrastructure
+    tag: assisted-test-infra
+  builder:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.14"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |
```

Comment:
Would [this](https://github.com/rh-ecosystem-edge/recert/blob/main/Dockerfile.prow) help?

Reply:
Awesome, I had this change in my TODO list! I'll try it and either add the respective change here or in a separate PR (because I would like to also change the `main` branch configuration as well).